scripts/sync_rumba_updates.sh

```diff
@@ -0,0 +1,12 @@
+updates_host=${UPDATES_HOST?};
+api_host=${RUMBA_HOST?};
+
+most_recent_update=`curl --fail $updates_host/rumba-bcd-updates/bcd-updates.json | jq '[.browsers[].releases[] | select(.status == "current")] | max_by(.release_date) | .release_date'`
+most_recent_api_release=`curl --fail $api_host/api/v2/updates/ | jq '.data[0].release_date'`
+if [ $(date -d $most_recent_update +%s) -gt $(date -d $most_recent_api_release +%s) ]; 
```

Comment:
Looks like this fails, see: https://github.com/mdn/bcd-utils/actions/runs/4317279119/jobs/7534154507#step:3:21

```
...
date: invalid date ‘"2023-02-14"’
date: invalid date ‘"2023-02-14"’
"2023-02-14" is more recent than "2023-02-14"
```

Reply:
And `jq -r` should do the trick, see: https://stackoverflow.com/a/44656583

Reply:
Fixed via f4f8617589623902278a150817f334d0abf81d3c.

Now it looks like expected: `2023-02-14 is not more recent than 2023-02-14. Finished!`

https://github.com/mdn/bcd-utils/actions/runs/4317320441/jobs/7534254246#step:3:20